NSS-Day-Cohort-49/tabloid-cli-spiny-crabs
# Mat author tags

#### Changes Made
1. Updated Author detail manager with conditional statement
?
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout mat-authorTags
    ```
2. Open a new Terminal tab (?T) and navigate to the server directory.
3. Test app functionality.
    > Navigate to author menu
    > Select author details
    > Select an author
    > Add tag to author
    > Attempt to add same tag, should not work
    > Remove tag from author
    > view tags to see that they appear correctly
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
